### DISCUSSION TITLE
I gave a wrong tag name and want to change it

### DISCUSSION THREAD

**Discussion by crisleicon** [community/community#130930] [Discussions]
### Select Topic Area

Question

### Body

I created wrongly a tag name in my repo as v9.1, then I gave its Release Title as v9.0 and in the Description I specified that this tag belongs to 9.0 version. 
Now the link to access the tag of my repo says like:  my_repo_url/releases/v9.1 
I was wondering if there's a way to change this link to: my_repo_url/releases/v9.0

**Comment by alexisseurin** [comment]
It removes the old tag and replacing it with the new one.

`git push origin [new_tag_name] :[old_tag_name]`

`git push origin v1.1 :v1.0`

**Reply by crisleicon** [reply]
Thanks for you help. I just tried that out
git push origin v9.0 :v9.1
error: unable to delete 'v9.1': remote ref does not exist
error: failed to push some refs to 'my_repo_url/app.git'
Any thought on what could be?

**Reply by alexisseurin** [reply]
git push origin v9.1 :v9.0

**Reply by alexisseurin** [reply]
Another example:
```
git tag v9.1 v9.0 # Create a new local tag named v9.1 from tag v9.0
git tag -d v9.0 # Delete local tag v9.0 
git push origin v9.1 :v9.0 # Push v9.1 and delete tag v9.0 on origin
```

**Comment by mfm-builder-001** [comment]
There is a way to update the URL to reflect the correct version number, even if the tag name is incorrect. Here's how you can do it:

1. Delete the existing tag:
In your local repository, run the following command to delete the tag:
`git tag -d v9.1`
Then, push the tag deletion to the remote repository:
`git push origin :refs/tags/v9.1`
2. Create a new tag with the correct version number:
In your local repository, create a new tag with the correct version number:
`git tag -a v9.0 -m "Release v9.0"`
Push the new tag to the remote repository:
`git push origin v9.0`
3. Update the release on GitHub (or the platform you're using):
Go to the "Releases" section of your GitHub repository.
Find the release with the incorrect version number (v9.1) and click on the "Edit" button.
Update the "Tag version" field to "v9.0".
Update the "Release title" to "v9.0" (if it's not already).
Update the "Description" to reflect the correct version number.
Click "Update release" to save the changes.
After following these steps, the URL for the release should now be updated to my_repo_url/releases/v9.0, as you desired.

**Comment by crisleicon** [comment]
Thank you all guys, I really appreciate your help :)  
I run the command:  git push --delete origin v9.1  and then I have created the tag v9.0 with the same commits and finally associated the release tag v9.0.
